src/op/tensorize.cc

```diff
@@ -187,7 +187,8 @@ class TensorIntrinMatcher final : public IRMutator {
             const Stage& stage,
             const std::unordered_map<IterVar, Range>& out_dom,
             const std::unordered_map<Tensor, Array<Range> >& in_region,
-            const TensorIntrin& intrin) {
+            const TensorIntrin& intrin,
+            Map<Var, Range>* compute_intrin_iter_space = nullptr) {
```

Comment:
Is additional pass of map necessary? or can we simply set it to be the corresponding domain?

Reply:
we're populating the map in Init and we need to use it in VerifyTensorizeBody which calls MatchTensorizeBody (which calls Init). That's why the chain of passing the map. Am I missing something?

Reply:
I see, I am a bit confused by the default argument value(nullptr). For internal function, we should always encourage passing in the values by default.

Reply:
instead of the default argument

Reply:
The only reason I made it an optional argument was that there was another call to MatchTensorizeBody and that was in `test.op.MatchTensorizeBody` Python API definition. 
Should I go ahead and make it a regular (non-optional) argument and pass the var range info to the other invocation as well?

Reply:
yes, this is internal API test, so we can do that